app/views/drivers/_form.html.slim

```diff
@@ -0,0 +1,35 @@
+= form_with model: @driver do |f|
```

Comment:
I suggest not use the instance variable directly.
You can set getter for this variable in the controller:
```
helper_mehtod :driver
def driver
  @driver
end
```

So in case of removing or renaming it in the controller, you'll see a variable not defined exception here instead of working with nil values

Reply:
And now you should use this getter in the view instead of an instance variable
```
= form_with model: driver do |f|
```

Reply:
if we update to use a helper, how does this work when we need to return the driver after an error?
would it just use the updated `@driver` value that we set when we call the helper?

Reply:
@benrobinson-ms
Yes. This change not in scope of the fix an error. Just some tricks for the coding culture to use less instance variable in the view as they are defined in the controller and it would be hard to clean all views if controller changes.

E.g. if in the future instance variable `@driver` changes to `@current_driver`
You need to find the usage of this variable in each view. And in some cases it could fail silently as the instance variable is present by default and it is nil.

In case you use the helper method in view. When you change the name from `driver` to `current_driver` it will fail with `Undefined variable`